src/test/java/de/friday/test/support/rules/dsl/gosu/GosuRuleSpecification.java

```diff
@@ -17,17 +17,22 @@
 package de.friday.test.support.rules.dsl.gosu;
 
 import de.friday.sonarqube.gosu.plugin.rules.BaseGosuRule;
-import de.friday.test.support.rules.dsl.specification.RuleSpecification;
 import de.friday.test.support.rules.dsl.specification.IssueSpecification;
+import de.friday.test.support.rules.dsl.specification.RuleSpecification;
 import de.friday.test.support.rules.dsl.specification.SourceCodeFile;
+import org.sonar.api.batch.sensor.issue.Issue;
+
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
-import org.sonar.api.batch.sensor.issue.Issue;
 
 public class GosuRuleSpecification implements RuleSpecification {
     private SourceCodeFile sourceCodeFile;
     private Class<? extends BaseGosuRule> checkClass;
 
+    private final Map<String, String> parameters = new HashMap<>();
```

Comment:
How about ruleProperties? 
This way it will follow the same nomenclature of the Sonar Plugin API `@RuleProperty`.